core/network/src/main/java/com/umcspot/spot/network/TokenAuthenticator.kt

```
@@ -0,0 +1,117 @@
+            }
+
+            val refreshToken = tokenData.refreshToken
+            if (refreshToken.isBlank()) {
+                clearAuthData()
+                return null
+            }
+
+            val refreshResponse = runBlocking {
+                try {
+                    tokenRefreshService.refreshTokenData(refreshToken)
+                } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)